### DISCUSSION TITLE
player.changeSprite is not a function

### DISCUSSION THREAD

**Discussion by prixsri** [replit/kaboom#603] [Q&A] [answered]
I am making a very basic maze game and was trying to change direction of player but .changeSprite() doesn't work

I have 4 sprite for each direction
` 
 keyDown('up', () => {
    player.changeSprite('link-up')
    p.move(0, -SPEED)
  })
`
can somebody please help

**Comment by lajbel** [comment, marked as answer]
yeah, it's removed in favor of `obj.use(comp)`, in this case

`obj.use(sprite("link-up"))`
